src/models/TestExecutable.js

```diff
@@ -123,17 +123,30 @@ class TestExecutable {
       return FINISHED_EXECUTION;
     })();
     let result = await Promise.race([timeoutTimer, testExecution]);
-    //TIMING
+    // TIMING
     const t1 = performance.now();
     const elapsedTime = t1 - t0;
 
-    //OVERALL SUCCESS
+    // OVERALL SUCCESS
     let success = _actionReports.filter((a) => a.success == false).length == 0 && result === FINISHED_EXECUTION;
+    let _finalReport = _actionReports;
+
+    if (result === TIMEOUT) {
+      // Add a failed step for the test timeout limit and inform the user that subsequent steps 
+      // have been skipped
+      _finalReport.push({
+        action: actions[_actionReports.length].action,
+        success: false,
+        shortSummary: `Test exceeded timeout limit of ${timeoutSeconds}s. Subsequent steps skipped.`,
```

Comment:
In your commit, you use the `stepTimeoutSeconds` instead of the `timeoutSeconds`. These values are a bit confusing. I think we could possibly rename `stepTimeoutSeconds` to `requestTimeout`, in any case, it only affects the HTTP  step and the bug here was related to the `timeoutSeconds` value. So I changed that to be reflected in the error message.

Reply:
I also change the language here a bit so its clear all other steps are skipped.